packages/main/src/plugin/experimental-feature-feedback-handler.ts

```
@@ -203,7 +203,7 @@ export class ExperimentalFeatureFeedbackHandler {
 
       this.messageBox
         .showMessageBox({
-          title: `Share Your Feedback`,
+          title: `Share Feedback`,
```

Biome (2.4.11)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)